spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories

```diff
@@ -20,4 +19,10 @@ org.springframework.cloud.client.discovery.EnableDiscoveryClient=\
 org.springframework.cloud.netflix.eureka.EurekaDiscoveryClientConfiguration
 
 org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker=\
-org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
\ No newline at end of file
+org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
+
+org.springframework.cloud.netflix.metrics.servo.EnableServo=\
```

Comment:
Quick question? Do you want these to be auto-configured, or only enabled via `@Enable*` annotations. I'm close to merging this :-)